numerics/poisson_series.hpp

```diff
@@ -299,6 +301,18 @@ class PiecewisePoissonSeries {
   // t must be in the interval [t_min, t_max].
   Value operator()(Instant const& t) const;
 
+  // Returns the Fourier transform of this Poisson series on [t_min, t_max].
+  // When evaluated at a given frequency, the Fourier transform is computed by
+  // Gauss-Legendre quadrature on each subinterval, where the number of points
+  // is chosen assuming that the periods of periodic terms are all large
+  // compared to the subintervals.
+  // If apodization is desired, the |*this| should be multiplied by an
+  // apodization function, and |FourierTransform| should be called on the
+  // product.
+  // |*this| must outlive the resulting function.
+  std::function<Complexification<Value>(AngularFrequency const&)>
```

Comment:
I'd favour giving a name to this type.

Reply:
Also, this file should include `functional`.

Reply:
> I'd favour giving a name to this type.

meh.

> Also, this file should include `functional`.

Done.